src/client.ts

```diff
@@ -946,4 +946,12 @@ export class SpvWalletClient {
       metadata,
     });
   }
+
+  /**
+   * Retrieves the shared configuration from the server.
+   * @returns {SharedConfig} A promise that resolves to the shared configuration.
+   */
+  async GetSharedConfig(): Promise<SharedConfig> {
+    return await this.http.request(`shared-config`, 'GET');
```

Comment:
During tests we should check whether it works with an `adminKey` as good as with "standard" `xPrivs`.